pkgs/tools/package-management/protontricks/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, python3Packages, fetchFromGitHub
+, wineWowPackages, winetricks, gnome3
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "protontricks";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "Matoking";
+    repo = "protontricks";
+    rev = version;
+    sha256 = "1q5pn69d8zl7ndd22nrc5i5n5lw29jknv8a1n6cy365nqglha5yq";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ vdf ];
+
+  # The wine install shipped with Proton must run under steam's
+  # chrootenv, but winetricks and zenity break when running under
+  # it. See https://github.com/NixOS/nix/issues/902.
+  #
+  # The current workaround is to use wine from nixpkgs
+  makeWrapperArgs = [
+    "--set STEAM_RUNTIME 0"
+    "--set-default WINE ${wineWowPackages.minimal}/bin/wine"
+    "--set-default WINESERVER ${wineWowPackages.minimal}/bin/wineserver"
+    "--prefix PATH : ${lib.makeBinPath [ winetricks gnome3.zenity ]}"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A simple wrapper for running Winetricks commands for Proton-enabled games";
+    homepage = https://github.com/Matoking/protontricks;
+    repositories.git = https://github.com/Matoking/protontricks.git;
```

Comment:
please don't add this attribute, see https://github.com/NixOS/nixpkgs/issues/61435

Reply:
Done. I modified the commits and force pushed the changes. Would it be better to add a new commit instead for future reference?

Reply:
I'd say for @aanderse review it would be a good idea to do that.
That way we can review a new set of commits that has those fixes instead of reviewing
from scratch again.

Reply:
Please don't let me stand in the way of you reviewing this as well @worldofpeace :smile: